packages/AlgernonTests.package/ALGTestUI.class/instance/testMouseItemSelection.st

```diff
@@ -0,0 +1,9 @@
+testing
+testMouseItemSelection
+	| hand position moveEvent |
+	self runQuery: 'ALGCore'.
+	hand:= self algInstance world activeHand.
+	position:= self algInstance itemList submorphs third bounds center.
+	moveEvent:=MouseButtonEvent new setType: #mouseMove position: position which: 0 buttons: 0 hand: hand stamp: nil.
```

Comment:
Same formatting and code here. Maybe we could have a helper/fixture for creating these events, that sets defaults and you just overwrite what you need?